guides/leave_space_mutation.md

```diff
@@ -0,0 +1,50 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'leave-space-mutation'
+is: 'experimental'
+---
+
+# Leave Space
+
+## Concepts
+
+The _leaveSpace_ mutation allows you to leave a workspace, where applicable, with the Space ID. Currently this only applicable to spaces you are currently invited to.
+In this scenario _leaveSpace_ is used to reject the invite. 
+
+## Schema
+
+### Leave Space Mutation
+
+
+
+```graphql
+type MutationRoot {
+  ...
+  leaveSpace(input: SpaceLeaveInput!): SpaceLeaveMutation
+}
+
+type SpaceLeaveMutation {
+  successful: Boolean!
+}
+
+type SpaceLeaveInput {
+  id: String!
+}
+```
+
+## Example Request
+
+~~~~
+Method: POST
+URL: https://api.watsonwork.ibm.com/graphql
+Headers: 'Content-Type: application/graphql' , 'x-graphql-view: PUBLIC, EXPERIMENTAL'
+Body:
+{
+  mutation {
+    leaveSpace(input: {id: "4aefed18-0dc0-4314-ab73-7cffeef903f1"}) {
```

Comment:
don't use actual IDs

Reply:
What do you use elsewhere? A random UUID is a good example.

Reply:
Actually I just noticed that our UUID format doesn't have any hyphens, so I will update that if it's kept.